nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/state/navigation/navigation.reducer.ts

```diff
@@ -28,7 +28,14 @@ export const navigationReducer = createReducer(
     initialState,
     on(pushBackNavigation, (state, { backNavigation }) => {
         return produce(state, (draftState) => {
-            draftState.backNavigations.push(backNavigation);
+            if (draftState.backNavigations.length > 0) {
```

Comment:
Looks like there is still a bug in here:

![Kapture 2024-05-23 at 21 59 20](https://github.com/apache/nifi/assets/6797571/bb4db327-47ae-48e0-9704-94b8580442f0)